client/containers/Pub/SpubHeader/PreviewTab/PreviewTab.tsx

```diff
@@ -38,7 +38,7 @@ const PreviewTab = (props: Props) => {
 							<span className="text-wrapper">{pubData.title}</span>
 						</h1>
 						<h3 className="description pub-header-themed-secondary">
```

Comment:
btw I think we should write `{pubData.description && <h3> ...` so that this box doesn't appear when there is no description.